lib-java/README.md

```diff
@@ -7,7 +7,7 @@
 
 ### DESCRIPTION
 
-This <a target="_blank" href="https://github.com/signalfx/integrations/tree/release/lib-java">repository</a> contains libraries for instrumenting Java applications and reporting metrics to SignalFx, using Codahale Metrics.
+This <a target="_blank" href="https://github.com/signalfx/signalfx-java">repository</a> contains libraries for instrumenting Java applications and reporting metrics to SignalFx, using Codahale Metrics.
```

Comment:
I can't, of course, confirm whether this is the right link, so I will let others confirm that. However, while we are here, let's change "SignalFx" to "Splunk Infrastructure Monitoring". :-)

Reply:
Done. Nice catch.